paper-text-field.html

```diff
@@ -115,9 +115,8 @@
         pastedText = event.clipboardData.getData('text/plain');
       }
       // Allow the pasted text and update the labels.
-      document.execCommand('insertText', true, pastedText);
+      document.execCommand('insertText', false, pastedText);
       this._onInput();
-      event.preventDefault();
```

Comment:
Is this not needed to make sure the text isn't inserted by `execCommand` and then also inserted by the paste? My knowledge of `execCommand` is pretty much zero.